packages/wrangler/src/api/mixedMode/index.ts

```diff
@@ -0,0 +1,35 @@
+import path from "node:path";
+import { getBasePath } from "../../paths";
+import { startWorker } from "../startDevWorker";
+import type { StartDevWorkerInput, Worker } from "../startDevWorker/types";
+
+type MixedModeSession = Pick<Worker, "ready" | "setConfig"> & {
+	["mixedModeConnectionString"]: MixedModeConnectionString;
+};
+
+declare const __brand: unique symbol;
+export type MixedModeConnectionString = Awaited<Worker["url"]> & {
+	[__brand]: "MixedModeConnectionString";
+};
+
+export async function startMixedModeSession(
+	bindings: StartDevWorkerInput["bindings"]
+): Promise<MixedModeSession> {
+	const proxyServerWorkerEntrypoint = path.resolve(
+		getBasePath(),
+		"templates/mixedMode/proxyServerWorker.ts"
+	);
+	const { ready, setConfig, url } = await startWorker({
+		entrypoint: proxyServerWorkerEntrypoint,
+		dev: {
+			remote: true,
+		},
+		bindings,
+	});
+
+	return {
+		ready,
+		setConfig,
```

Comment:
This shouldn't be the actual `setConfig()`, it should be a function that only accepts the `bindings` subset of `setConfig()`

Reply:
oh right! good call, sorry I missed that! 😓👍

Reply:
addressed 🙂